Makefile

```diff
@@ -37,7 +15,6 @@ DOCKER_ENVS := \
 	-e DOCKER_CLIENTONLY \
 	-e DOCKER_DEBUG \
 	-e DOCKER_EXPERIMENTAL \
-	-e DOCKERFILE \
```

Comment:
(Where this way, we'll choose which `Dockerfile` to steal that line from as-necessary dynamically based on our platform regardless of which `Dockerfile` we're building from.)